addons/mail/static/src/core/web/message_model_patch.js

```diff
@@ -6,7 +6,7 @@ import { patch } from "@web/core/utils/patch";
 const messagePatch = {
     /** @param {import("models").Thread} thread the thread where the message is shown */
     canReplyAll(thread) {
-        return this.canForward(thread) && !this.is_note;
+        return this.canForward(thread) && !this.isNote;
```

Comment:
Again negation on note looks confusing, isn't it intended to target discussions specifically?